src/usr/local/emhttp/plugins/enhanced.log/data.php

```
@@ -153,4 +153,24 @@
         ->withStatus(200);
 });
 
+$app->get("{$prefix}/locales", function (Request $request, Response $response, $args) {
```

PHPStan (2.1.15)

Parameter $request of anonymous function has invalid type Psr\Http\Message\ServerRequestInterface.


(class.notFound)